this is the notes for ValidationErrors

for this lecture we are going to be looking into the error object 
given to us from our catch block